lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgid "Was filament unload successful?"
+msgstr "Lyckades filamentutmatningen?"
+
+# MSG_SELFTEST_WIRINGERROR c=18
+#: messages.c:98
+msgid "Wiring error"
+msgstr "Kabelfel"
+
+# MSG_WIZARD c=17
+#: ultralcd.cpp:5771
+msgid "Wizard"
+msgstr "Guide"
+
+# MSG_XYZ_DETAILS c=18
+#: ultralcd.cpp:2039
+msgid "XYZ cal. details"
+msgstr "XYZ kal. detaljer"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_FITTING_FAILED c=20 r=8
+#: messages.c:18
+msgid "XYZ calibration failed. Please consult the manual."
+msgstr "XYZ-kalibrering misslyckades. Se bruksanvisningen."
+
+# MSG_YES c=3
+#: messages.c:123
+msgid "Yes"
+msgstr "Ja"
+
+# MSG_WIZARD_QUIT c=20 r=8
+#: messages.c:120
+msgid "You can always resume the Wizard from Calibration -> Wizard."
+msgstr "Du kan alltid ateruppta guiden fran Kalibrering -> Guide."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_SKEW_EXTREME c=20 r=8
+#: ultralcd.cpp:3743
+msgid "XYZ calibration all right. Skew will be corrected automatically."
+msgstr "XYZ-kalibrering \xe1r ok. Skevhet kommer att korrigeras automatiskt."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_SKEW_MILD c=20 r=8
+#: ultralcd.cpp:3740
+msgid "XYZ calibration all right. X/Y axes are slightly skewed. Good job!"
+msgstr "XYZ-kalibrering ok. X/Y-axlarna \xe1r mycket lite skeva. Bra jobbat!"
+
+# MSG_TIMEOUT c=12
+#: messages.c:157
+msgid "Timeout"
+msgstr "Timeout"
+
+# MSG_X_CORRECTION c=13
+#: ultralcd.cpp:5086
+msgid "X-correct:"
+msgstr "X-korrektion:"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_PERFECT c=20 r=8
+#: ultralcd.cpp:3737
+msgid "XYZ calibration ok. X/Y axes are perpendicular. Congratulations!"
+msgstr "XYZ-kalibrering ok. X/Y-axlarna ar vinkelr\xe1ta. Grattis!"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_BOTH_FAR c=20 r=8
+#: ultralcd.cpp:3721
+msgid "XYZ calibration compromised. Front calibration points not reachable."
+msgstr "XYZ-kalibrering komprometterad. Fr\xe1mre kalibreringspunkter kan inte nas."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_RIGHT_FAR c=20 r=8
+#: ultralcd.cpp:3724
+msgid "XYZ calibration compromised. Right front calibration point not reachable."
+msgstr "XYZ-kalibrering komprometterad. H\xefger fr\xe1mre kalibreringspunkter kan inte nas."
+
+# MSG_LOAD_ALL c=17
+#: ultralcd.cpp:6167
+msgid "Load all"
+msgstr "Ladda alla"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_POINT_NOT_FOUND c=20 r=6
+#: ultralcd.cpp:3703
+msgid "XYZ calibration failed. Bed calibration point was not found."
+msgstr "XYZ-kalibrering misslyckades. B\xe1ddkalibreringspunkterna hittades inte."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_BOTH_FAR c=20 r=6
+#: ultralcd.cpp:3709
+msgid "XYZ calibration failed. Front calibration points not reachable."
+msgstr "XYZ-kalibrering misslyckades. Fr\xe1mre kalibreringspunkter kan inte nas."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_RIGHT_FAR c=20 r=6
+#: ultralcd.cpp:3712
+msgid "XYZ calibration failed. Right front calibration point not reachable."
+msgstr "XYZ-kalibrering misslyckades. H\xefger fr\xe1mre kalibreringspunkt kan inte nas."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.